spec/support/api/schemas/songs.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "http://jsonschema.net",
+  "type": "object",
+  "properties": {
+    "songs": {
+      "id": "http://jsonschema.net/songs",
+      "type": "array",
+      "items": {
+        "id": "http://jsonschema.net/songs/0",
+        "type": "object",
+        "properties": {
+          "id": {
+            "id": "http://jsonschema.net/songs/0/id",
+            "type": "string"
+          },
+          "name": {
+            "id": "http://jsonschema.net/songs/0/name",
+            "type": "string"
+          },
+          "artist": {
+            "id": "http://jsonschema.net/songs/0/artist",
+            "type": "string"
+          },
+          "spotify_uri": {
+            "id": "http://jsonschema.net/songs/0/spotify_uri",
+            "type": "string"
+          },
+          "created_at": {
+            "id": "http://jsonschema.net/songs/0/created_at",
+            "type": "string"
+          },
+          "updated_at": {
+            "id": "http://jsonschema.net/songs/0/updated_at",
+            "type": "string"
+          }
+        },
+        "required": [
+          "id",
+          "name",
+          "artist",
+          "spotify_uri",
+          "created_at",
+          "updated_at"
+        ]
+      },
+      "required": [
+        "0"
+      ]
+    }
+  },
+  "required": [
+    "songs"
+  ]
+}
```

Comment:
It's hard to see in the editor that it might be missing until pushed the GitHub.